pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -543,6 +543,78 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 			},
 		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryBasicUserRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("imagestreams", "images"),
+				},
+				{
+					Verbs:     sets.NewString("create", "list"),
+					Resources: sets.NewString("projectrequests"),
+				},
+				{
+					Verbs:     sets.NewString("list"),
+					Resources: sets.NewString("projects", "subjectaccessreview", "localsubjectaccessreview"),
+				},
+			},
+		},
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: RegistryAdminRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				{
+					Verbs:     sets.NewString("create", "delete", "list", "patch", "update", "watch"),
+					Resources: sets.NewString("imagestreams", "images"),
+				},
+				{
+					Verbs:     sets.NewString("create", "list"),
+					Resources: sets.NewString("projectrequests"),
+				},
+				{
+					Verbs:     sets.NewString("list", "watch"),
+					Resources: sets.NewString("users", "groups"),
```

Comment:
These are cluster level resources, which means that binding this role inside of a namespace won't have any affect.  Since this is intended to a namespace level role, we don't want to add these permissions here.  Remove